ovos_number_parser/numbers_pt.py

```
@@ -1,503 +1,533 @@
-            if count + 2 < len(aWords):
-                next_next_word = aWords[count + 2]
+    # Use variant-specific dictionaries
+    numbers_map = _NUMBERS_BR if variant == PortugueseVariant.BR else _NUMBERS_PT
+    scales_map = _SCALES[Scale.SHORT if short_scale else Scale.LONG][variant]
+
+    clean_text = text.lower().replace('-', ' ')
+    tokens = [t for t in clean_text.split() if t not in ['e', 'uma', 'um']]
+
+    result = 0
+    current_number = 0
+    scale_multiplier = 1
```

Ruff (0.12.2)

Local variable `scale_multiplier` is assigned to but never used

Remove assignment to unused variable `scale_multiplier`

(F841)